### PR TITLE
Updade french liblxqt lximage-qt pcmanfm-qt qtermwidget qterminal lxq…

### PR DESCRIPTION
**Pull request by paiiou**
…t-panel

### CODE REVIEW

**Comment by paiiou** [comment]
The below failure appears to be caused by missing CMakeList files. It does not therefore appear to be the result of translations

**Comment by paiiou** [comment]
After checking, I discovered that several files were missing in my 'pull request'
This is the command 'git checkout -b transl5_fr' which badly worked in my local repository
It is now corrected.

File: liblxqt/liblxqt_fr.ts [hunk 1]
```diff
@@ -2,158 +2,187 @@
...
+    <message>
+        <location filename="../configdialog/lxqtconfigdialog.ui" line="14"/>
+        <source>Dialog</source>
+        <translation>Dialogue</translation>
+    </message>
+</context>
+<context>
     <name>LXQt::MessageBox</name>
     <message>
-        <location filename="../../lxqtpowermanager.cpp" line="72"/>
```
Comment (palinek) [inline liblxqt/liblxqt_fr.ts]: Why did you change all the paths?

**Review by palinek — CHANGES_REQUESTED** [review]

File: liblxqt/liblxqt_fr.ts (hunk 1 above)
Comment (paiiou) [reply liblxqt/liblxqt_fr.ts]: Of course, the change was not voluntary. It is lupdate which determines the paths. Which means I was not in the right directory when I used lupdate. But I would say that there are many pitfalls for translators.
I corrected for the 4 packages

**Comment by paiiou** [comment]
As stated on April 3, I corrected the different paths.
I should point out that the correction was made based on models such as liblxqt.ts

**Comment by palinek** [comment]
> I should point out that the correction was made based on models such as liblxqt.ts

No, the paths are different, e.g.:
```diff
diff liblxqt/liblxqt.ts liblxqt/liblxqt_fr.ts 
3c3
< <TS version="2.1">
---
> <TS version="2.1" language="fr">
7c7
<         <location filename="../../configdialog/lxqtconfigdialog.ui" line="14"/>
---
>         <location filename="../configdialog/lxqtconfigdialog.ui" line="14"/>
9c9
<         <translation type="unfinished"></translation>
---
>         <translation>Dialogue</translation>
15c15
<         <location filename="../../lxqtpowermanager.cpp" line="70"/>
---
>         <location filename="../lxqtpowermanager.cpp" line="72"/>
17c17
<         <translation type="unfinished"></translation>
---
>         <translation>Erreur du gestionnaire d&apos;énergie de LXQt</translation>
20c20
<         <location filename="../../lxqtpowermanager.cpp" line="70"/>
---
>         <location filename="../lxqtpowermanager.cpp" line="72"/>
22c22
<         <translation type="unfinished"></translation>
---
>         <translation>Échec de la mise en hibernation.</translation>
28c28
<         <location filename="../../lxqtnotification.cpp" line="152"/>
---
>         <location filename="../lxqtnotification.cpp" line="152"/>
30c30
<         <translation type="unfinished"></translation>
---
>         <translation>Notifications de repli</translation>
36c36
<         <location filename="../../lxqtpowermanager.cpp" line="112"/>
---
>         <location filename="../lxqtpowermanager.cpp" line="114"/>
38c38
<         <translation type="unfinished"></translation>
---
>         <translation>Hiberner</translation>
41c41
<         <location filename="../../lxqtpowermanager.cpp" line="119"/>
---
>         <location filename="../lxqtpowermanager.cpp" line="121"/>
43c43
<         <translation type="unfinished"></translation>
---
>         <translation>Suspendre</translation>
46c46
<         <location filename="../../lxqtpowermanager.cpp" line="126"/>
---
>         <location filename="../lxqtpowermanager.cpp" line="128"/>

...

145,149c147,149
<         <location filename="../../lxqtscreensaver.cpp" line="83"/>
<         <location filename="../../lxqtscreensaver.cpp" line="89"/>
<         <location filename="../../lxqtscreensaver.cpp" line="96"/>
<         <source>Screen Saver Activation Error</source>
<         <translation type="unfinished"></translation>
---
>         <location filename="../lxqtscreensaver.cpp" line="175"/>
>         <source>An error occurred starting screensaver. Unknown error - undocumented return value from xdg-screensaver: %1.</source>
>         <translation>Echec de démarrage de l&apos;économiseur d&apos;écran. Erreur inconnue - retour d&apos;une valeur non documentée de l&apos;économiseur xdg : %1.</translation>
152,154c152,154
<         <location filename="../../lxqtscreensaver.cpp" line="84"/>
<         <source>An error occurred starting screensaver. Ensure you have xscreensaver installed and running.</source>
<         <translation type="unfinished"></translation>
---
>         <location filename="../lxqtscreensaver.cpp" line="156"/>
>         <source>An error occurred starting screensaver. Syntax error in xdg-screensaver arguments.</source>
>         <translation>Une erreur est survenue lors du démarrage de l&apos;écran de veille. Les arguments de xdg-screensaver contiennent des erreurs de syntaxe.</translation>
157,159c157,159
<         <location filename="../../lxqtscreensaver.cpp" line="90"/>
<         <source>An error occurred starting screensaver. Action &apos;activate&apos; failed. Ensure you have xscreensaver installed and running.</source>
<         <translation type="unfinished"></translation>
---
>         <location filename="../lxqtscreensaver.cpp" line="162"/>
>         <source>An error occurred starting screensaver. Ensure you have xscreensaver installed and running.</source>
>         <translation>Une erreur est survenue lors du démarrage de l&apos;écran de veille. Assurez-vous que screensaver est  installé et actif.</translation>
162,164c162,164
<         <location filename="../../lxqtscreensaver.cpp" line="97"/>
<         <source>An error occurred starting screensaver. Unknown error - undocumented return value from xdg-screensaver: %1.</source>
<         <translation type="unfinished"></translation>
---
>         <location filename="../lxqtscreensaver.cpp" line="168"/>
>         <source>An error occurred starting screensaver. Action &apos;activate&apos; failed. Ensure you have xscreensaver installed and running.</source>
>         <translation>Une erreur est survenue lors du démarrage de l&apos;écran de veille. L&apos;action &apos;activate&apos; a échoué. Assurez-vous que screensaver est  installé et actif.</translation>
170,171c170,171
<         <location filename="../../lxqtpower/lxqtpowerproviders.cpp" line="91"/>
<         <location filename="../../lxqtpower/lxqtpowerproviders.cpp" line="141"/>
---
>         <location filename="../lxqtpower/lxqtpowerproviders.cpp" line="87"/>
>         <location filename="../lxqtpower/lxqtpowerproviders.cpp" line="137"/>
173c173
<         <translation type="unfinished"></translation>
---
>         <translation>Erreur du gestionnaire d&apos;énergie de LXQt</translation>
176,177c176,177
<         <location filename="../../lxqtpower/lxqtpowerproviders.cpp" line="92"/>
<         <location filename="../../lxqtpower/lxqtpowerproviders.cpp" line="142"/>
---
>         <location filename="../lxqtpower/lxqtpowerproviders.cpp" line="88"/>
>         <location filename="../lxqtpower/lxqtpowerproviders.cpp" line="138"/>
179c179
<         <translation type="unfinished"></translation>
---
>         <translation>L&apos;interface QDBus est invalide</translation>
182,183c182,183
<         <location filename="../../lxqtpower/lxqtpowerproviders.cpp" line="106"/>
<         <location filename="../../lxqtpower/lxqtpowerproviders.cpp" line="156"/>
---
>         <location filename="../lxqtpower/lxqtpowerproviders.cpp" line="102"/>
>         <location filename="../lxqtpower/lxqtpowerproviders.cpp" line="152"/>
185c185
<         <translation type="unfinished"></translation>
---
>         <translation>Erreur du gestionnaire d&apos;énergie de LXQt (appel de D-Bus)</translation>
```

**Comment by paiiou** [comment]
Indeed, the changes I made locally were not reflected on the remote repo. This is due to my poor knowledge of git. As a result, I checked all translations (with their path) and found other errors. Now this should be correct.

**Comment by palinek** [comment]
thanks

**Pull request merged**